internal/helper/loadbalancer.go

```diff
@@ -327,12 +327,23 @@ func parseLoadBalancerInfoMetric(
 		"isInternal":       strconv.FormatBool(lb.Spec.Options.InternalLB),
 		"tcpProxyProtocol": strconv.FormatBool(lb.Spec.Options.TCPProxyProtocol),
 		"externalIP":       "nil",
+		"tcpIdleTimeout":   "nil",
+		"udpIdleTimeout":   "nil",
+		"lokiEnabled":      strconv.FormatBool(lb.Spec.Options.LogForward.Enabled),
 	}
 
 	if lb.Status.ExternalIP != nil {
 		labels["externalIP"] = *lb.Status.ExternalIP
 	}
 
+	if lb.Spec.Options.TCPIdleTimeout != nil {
+		labels["tcpIdleTimeout"] = strconv.Itoa(*lb.Spec.Options.TCPIdleTimeout)
```

Comment:
does this also work with the duration? or lets merge this one first, rebase, and fix in the other pr :)

Reply:
I will update it in the other PR